rhine-terminal/rhine-terminal.cabal

```diff
@@ -0,0 +1,72 @@
+-- Initial rhine-gloss.cabal generated by cabal init.  For further
+-- documentation, see http://haskell.org/cabal/users-guide/
+
+name:                rhine-terminal
+version:             0.8.0
+synopsis:            Terminal backend for Rhine
+description:
+  This package provides an example of a `terminal` based program using rhine.
+license:             BSD3
+license-file:        LICENSE
+author:              Manuel Bärenz, Jun Matsushita
+maintainer:          programming@manuelbaerenz.de, jun@iilab.org
+-- copyright:
+category:            FRP
+build-type:          Simple
+extra-source-files:  ChangeLog.md
+extra-doc-files:     README.md
+cabal-version:       1.18
+
+source-repository head
+  type:     git
+  location: git@github.com:turion/rhine.git
+
+source-repository this
+  type:     git
+  location: git@github.com:turion/rhine.git
+  tag:      v0.8.0.0
+
+library
+  exposed-modules:
+    FRP.Rhine.Terminal
+  build-depends:       base         >= 4.11 && < 4.16
+                     , exceptions   >= 0.10.4
+                     , transformers >= 0.5
+                     , rhine        == 0.8.0.0
+                     , dunai        >= 0.6
+                     , terminal     >= 0.2.0.0
+                     , time         >= 1.9.3
+  hs-source-dirs:      src
+  default-language:    Haskell2010
+  ghc-options:         -Wall
```

Comment:
Can you have a look at https://github.com/turion/rhine/commit/c3ef9c6df7566a1eee2006eb9dab98f12323bff7 and adapt this accordingly?